.github/actions/prepare-mobile-build/action.yaml

```diff
@@ -4,9 +4,15 @@ description: Action to prepare environment for mobile build
 runs:
   using: composite
   steps:
-    - uses: ruby/setup-ruby@9669f3ee51dc3f4eda8447ab696b3ab19a90d14b # v1.144.0
-      with:
-        ruby-version: "2.7.7"
+    - name: ci/setup-ruby
+      shell: bash
+      run: |
+        brew update
+        brew install rbenv ruby-build
+        rbenv install $(cat .ruby-version)
```

Comment:
not sure about this, the runner for `macos-14-large` already comes with `Ruby 3.0.6p216` and the following versions are cached

2.7.8
3.0.6
3.1.4